### ISSUE TITLE
[Component] [input] [accessibility] Missing role on input limit length

### ISSUE THREAD

**Issue by chou666** [element-plus/element-plus#24674]
Bug Type: **`Component`**

## Environment

- Vue Version: `3.5.41`
- Element Plus Version: `2.14.3`
- Browser / OS: `Firefox / Ubuntu`
- Build Tool: `Vite`

### Related Component

- `el-input`

### Reproduction Link

[Docs](https://element-plus.org/en-US/component/input#limit-length)

### Steps to reproduce

Enter some text with vocal assistant

### What is Expected?

Remaining characters count must be read by vocal assistant

### What is actually happening?

Nothing

### Additional comments

Correction: Add a role="status" attribute around the label so that screen readers read it out
whenever it is updated (+ add hidden text “characters” within the same component so that the user
knows that #/120, for example, corresponds to the number of characters)
